test/suites/dev-tanssi-relay/ethereum-token-transfers/test_receive_container_native_tokens_from_eth.ts

```diff
@@ -0,0 +1,143 @@
+import "@tanssi/api-augment";
+
+import { beforeAll, describeSuite } from "@moonwall/cli";
+import { type ApiPromise, Keyring } from "@polkadot/api";
+import { generateEventLog, generateUpdate, mockAndInsertHeadData } from "utils";
+import { expectEventCount, STARLIGHT_VERSIONS_TO_EXCLUDE_FROM_CONTAINER_TRANSFERS } from "helpers";
+import type { KeyringPair } from "@moonwall/util";
+
+describeSuite({
+    id: "DTR1802",
+    title: "NativeContainerTokensProcessor: receive and forward container native tokens from Ethereum",
+    foundationMethods: "dev",
+
+    testCases: ({ it, context }) => {
```

Comment:
how hard is to generate failure tests here?

- Example: we recieve a non-registered token. 

If so how can we verify it is working as expected? I am assuming we can check for a non-sent event